src/Products/PlonePAS/tools/groupdata.py

```diff
@@ -132,7 +132,7 @@ def getGroup(self):
         bcontext = aq_base(parent)
         bcontainer = aq_base(aq_parent(aq_inner(self)))
         if bcontext is bcontainer or not hasattr(bcontext, 'getUserName'):
-            raise 'GroupDataError', "Can't find group data"
+            raise 'GroupDataError'("Can't find group data")
```

Comment:
This must be really old code. The old code gives a TypeError on Python 2.6 and higher.
Your new code gives "TypeError: 'str' object is not callable".
Probably best to define a `GroupDataError` class that inherits from `Exception`, and raise that here.